docs/user-guide/persistent-volumes/README.md

```diff
@@ -68,6 +68,29 @@ NAME      LABELS       CAPACITY      ACCESSMODES   STATUS      CLAIM     REASON
 pv0001    type=local   10737418240   RWO           Available 
 ```
 
+### Access Control
+
+If you have some existing underlying storage with ownership
```

Comment:
I'd start out more declaratory of what and why:

Storage configured with GID will only allow writing by pods using the same GID. Mismatched or missing GIDs will cause 'permission denied' errors.  Adding GID to a PersistentVolume allows Kubelet to automatically set the GID a the pod that requires it. No coordination between an admin and end user is required. Setting GID automatically is safe because this is only done for PersistentVolumes which are accessed through a PersistentVolumeClaim.  Users are restricted to their claims by namespace.